### PR TITLE
Translating about dialog on QT and updating copyright date

### PR DESCRIPTION
**Pull request by Oreonan**

### CODE REVIEW

File: Client/qtTeamTalk/settings.h [hunk 1]
```diff
@@ -26,12 +26,12 @@
...
 #include <QSettings>
 
-#define SETTINGS_VERSION                            "5.1"
+#define SETTINGS_VERSION                            "5.6"
 #define SETTINGS_GENERAL_VERSION                    "version"
 #define SETTINGS_GENERAL_VERSION_DEFAULT            "5.0"
 
 #define SETTINGS_GENERAL_NICKNAME                   "general_/nickname"
-#define SETTINGS_GENERAL_NICKNAME_DEFAULT           "NoName"
+#define SETTINGS_GENERAL_NICKNAME_DEFAULT           tr("NoName")
```
[the remark this answers is not in the document]
Comment (Oreonan) [reply Client/qtTeamTalk/settings.h:34]: Euh yes, but in which cpp file ? I did not find a match in any cpp file :)

[the remark this answers is not in the document]
Comment (bear101) [reply Client/qtTeamTalk/settings.h:34]: my mistake. Please merge now with master. I've added tr(SETTINGS_GENERAL_NICKNAME_DEFAULT).
You can remove tr() from settings.h

[the remark this answers is not in the document]
Comment (Oreonan) [reply Client/qtTeamTalk/settings.h]: I hope everything is good. I'm really not good at maintaining 2 branches at the same time :)
Isn't there something to add to the ts files for translating the default name?

[the remark this answers is not in the document]
Comment (bear101) [reply Client/qtTeamTalk/settings.h]: when I update the .ts files next time you should see it.

[the remark this answers is not in the document]
Comment (Oreonan) [reply Client/qtTeamTalk/settings.h]: OK, I'll watch it all :)

[the remark this answers is not in the document]
Comment (Oreonan) [reply Client/qtTeamTalk/settings.h]: @bear101 I've don't saw "noname" in ts file. It's normal ?

**Pull request merged**